app/assets/javascripts/lib/ui/content/annotations/replace.js

```diff
@@ -20,7 +20,8 @@ delegate(document, '.annotate.replacement', 'keypress', e => {
 // Pressing enter or spacebar in the contenteditable region shouldn't press the containing button
 delegate(document, '.annotate.replacement .text', 'keypress', e => {
   if (e.key=='Enter'||e.key==' '||e.keyCode==13||e.keyCode==32){
-    e.stopPropagation();
+    document.querySelector('[data-annotate-action=save-changes]').click()
+    e.target.blur();
```

Comment:
Hi! I saw the word "focus" go by on Slack, so I thought I'd pop by this PR. 

Calling `blur()` drops keyboard focus, which means that anybody navigating via a keyboard or assistive technology will have to start again from the top of the page after this code runs: not a great user experience, exactly like when making an annotation caused the screen to scroll all the way to the top. 

Instead of calling `blur()` to unset the keyboard focus entirely, you can call `focus()` on the element you'd like to set focus to.

Reply:
Also noticed: this code is listening for spacebar too (`e.key==' '`, `e.keyCode==32`); I'm guessing you still want people to be able to press spacebar without triggering that new click event?